crypto/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ zephyr_interface_library_named(nrfxlib_crypto)
 
 if (CONFIG_NRF_OBERON)
   set(OBERON_BASE ${CMAKE_CURRENT_SOURCE_DIR}/nrf_oberon)
-  set(OBERON_VER 3.0.3)
+  set(OBERON_VER 3.0.5)
```

Comment:
What happened to 304? :D

Reply:
I will make a documentary about it 10 years in the future :) Find people who have "heard about the release"